includes/class-wc-gateway-swedbank-pay-checkout.php

```diff
@@ -1,12 +1,23 @@
 <?php
+defined( 'ABSPATH' ) || exit;
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit;
-} // Exit if accessed directly
+use SwedbankPay\Checkout\WooCommerce\WC_Adapter;
+use SwedbankPay\Checkout\WooCommerce\WC_Background_Swedbank_Pay_Queue;
+use SwedbankPay\Checkout\WooCommerce\WC_Swedbank_Pay_Transactions;
+use SwedbankPay\Core\Core;
+use SwedbankPay\Core\OrderInterface;
+use SwedbankPay\Core\Log\LogLevel;
 
+class WC_Gateway_Swedbank_Pay_Checkout extends WC_Payment_Gateway {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class WC_Gateway_Swedbank_Pay_Checkout has 1994 lines of code. Current threshold is 1000. Avoid really long classes.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-checkout/pullRequest?prid=5510424)